src/svgs/logo.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
```

Comment:
Create `assets` folder in root of the project and all images there

Reply:
I once faced an issue, React wasn't allowing me use assets present outside the src folder.

Anyways, we don't need the folder anymore as I took the svg code from the `logo.svg` and added it to `svgs/logo.js` and exported it. Then whenever we want to use the svg we just add it to the JSX.
Something like:
```
import React from "react";
import logo from "../svgs/logo";
import Auth from "../modules/Auth";

const Navbar = () => (
  <nav>
    {logo}
    <Auth/>
  </nav>
);
export default Navbar;
```